client/pkg/pynuoca.py

```diff
@@ -0,0 +1,62 @@
+# (C) Copyright NuoDB, Inc. 2020  All Rights Reserved.
+#
+# Add the nuodb collection agent
+
+import os
+
+from client.package import Package
+from client.stage import Stage
+from client.artifact import PyPIMetadata, Artifact
+from client.utils import *
+
+class PyNuoCA(Package):
+    """Add the NuoDB Collection Agent"""
+
+    __PKGNAME = 'pynuoca'
+
+    class PyDep:
+        _file = ""
+        _version = ""
+        _basedir = ""
+
+    dependencies = {'aenum': PyDep(),
+                    'click': PyDep(),
+                    'elasticsearch': PyDep(),
+                    'python-dateutil': PyDep(),
+                    'PyYaml': PyDep(),
+                    'requests': PyDep(),
+                    'wrapt': PyDep(),
+                    'Yapsy': PyDep(),
+                    'kafka-python': PyDep(),
+                    'urllib3': PyDep(),
+                    'six': PyDep(),
+                    'certifi': PyDep(),
+                    'chardet': PyDep(),
+                    'idna': PyDep(),
+                    'pytz': PyDep(),
+                    }
```

Comment:
I'm not sure what these are for or where they're used?

Reply:
ah! Leftovers. Let me delete it